shared/src/business/useCases/serveCaseToIrs/generateDraftDocument.js

```diff
@@ -0,0 +1,35 @@
+const {
+  replaceBracketed,
+} = require('../../../business/utilities/replaceBracketed');
+
+/**
+ * @param {object} providers the providers object containing applicationContext, caseEntity, document, replacements
+ * @param providers.applicationContext api applicationContext
```

Comment:
You may be able to configure IDEA to output the type automatically when generating JSDoc.

Reply:
@jimlerza do you mean instead of `{object}`? It really is just an object with no named type. I did use WebStorm's auto-generation but had to change it. I never really considered that I might be able to customize what it generates. I might look into that.